lib/ansible_tower_client/v2/job_template_v2.rb

```diff
@@ -1,5 +1,5 @@
 module AnsibleTowerClient
-  class JobTemplate < BaseModel
+  class JobTemplateV2 < JobTemplate
```

Comment:
@bdunne - Since we're not taking advantage of the `/v2/` namespace is there any real reason to nest `JobTemplateV2` under that folder? 

On that note it would be nice to be able to use the `/v2/` namespace and not have to use the `JobTemplateV2` naming convention.